bigtable/CHANGELOG.md

```diff
@@ -4,6 +4,25 @@
 
 [1]: https://pypi.org/project/google-cloud-bigtable/#history
 
+## 0.31.0
+
+### Implementation Changes
+- Improve admin operation timeouts. ([#6010](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6010))
+
+### New Features
+
+### Dependencies
+
+### Documentation
+- Cloud Bigable readiness upgraded from alpha to beta.
+
+### Internal / Testing Changes
+- Preparing Cloud Bigtable for beta ([#6129](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6129))
+- Harden instance teardown against '429 Too Many Requests'. ([#6102](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6102))
+- Bigtable: prep docs for repo split. ([#6014](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6014))
```

Comment:
Remove "Bigtable: " prefix, make verb more clear, e.g. "Prepare docs for repo split."